.github/ISSUE_TEMPLATE/create-new-doctype-template.md

```diff
@@ -6,22 +6,51 @@ labels: templates
 assignees: ''
 
 ---
+# Title
+{Keep the title short, yet descriptive.
+Here are some examples:
+* Create a new {doc_type} template 
```

Comment:
We discussed this during our book club discussion. Since, we intend to use the book Docs for Developers as a reference material for templates. I am completely okay with changing doc type to content type.